src/utils/rootkeypackage_utils/inc/aduc/rootkeypackage_types.h

```diff
@@ -19,29 +19,37 @@
  */
 typedef enum tagADUC_RootKey_KeyType
 {
-    RSA, /**< RSA keytype. */
+    ADUC_RootKey_KeyType_INVALID = 0,
+    ADUC_RootKey_KeyType_RSA, /**< RSA keytype. */
 } ADUC_RootKey_KeyType;
 
 /**
  * @brief The root key SHA algorithms.
  */
 typedef enum tagADUC_RootKeyShaAlgorithm
```

Comment:
removing Sha from name and distinguishing between hashing vs signing algorithms, the latter are RS<256 | 384 | 512> and the former will be SHAversion from aduc/hash_utils.h